pkgs/amino/codec.go

```diff
@@ -780,7 +781,12 @@ func parseFieldOptions(field reflect.StructField) (skip bool, fopts FieldOptions
 func typeURLtoFullname(typeURL string) (fullname string) {
 	parts := strings.Split(typeURL, "/")
 	if len(parts) == 1 {
-		panic(fmt.Sprintf("invalid type_url \"%v\", must contain at least one slash and be followed by the full name", typeURL))
+		panic(
+			fmt.Sprintf(
+				"invalid type_url \"%v\", must contain at least one slash and be followed by the full name",
+				typeURL,
+			),
+		)
```

Comment:
:-1: 

previous version was way more readable

can be done in two independent lines, one for the sprintf, and one for the panic if long lines are really painful.